network/l2.c

```diff
@@ -98,9 +100,14 @@ typedef struct {
   uint16_t enabled_ports_mask;
   BmQueue evt_queue;
   BmTaskHandle task_handle;
-  L2LinkChangeCb link_change_cb;
+  LL link_change_cb;
 } BmL2Ctx;
 
+typedef struct {
+  uint8_t port;
```

Comment:
Please rename to `port_num` to make clear that it's one-indexed, as opposed to a `port_index` (zero-based) or `port_mask`.